acceptance/tests/resource/service/launchd_provider.rb

```diff
@@ -0,0 +1,82 @@
+test_name 'Mac OS X launchd Provider Testing'
+ 
+confine :to, {:platform => /osx/}, agents
```

Comment:
I've done this other places, it seems to filter the agents list, and if agents is empty it will skip the test.